core/src/main/scala-2.12/scala/collection/immutable/package.scala

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package scala.collection
+
+import scala.collection.immutable.ListMap
+
+package object immutable {
+  val SeqMap = ListMap
+  type SeqMap[K, +V] = ListMap[K, V]
+
+  val VectorMap = ListMap
+  type VectorMap[K, +V] = ListMap[K, V]
+}
```

Comment:
Here's a neat trick, we can use `SeqMap` and `VectorMap` within Chisel, but just alias them to regular old `ListMap` 🙂

Reply:
```scala
object Quicksort {
  def apply(a: Seq[Int]) = BubbleSort(a)
}
```